sources/packages/backend/apps/queue-consumers/src/processors/models/processors.models.ts

```diff
@@ -81,6 +82,24 @@ export class QueueProcessSummary {
       errors: this.errors.length ? this.errors : undefined,
     };
   }
+
+  /**
+   * Writes all the log entries.
+   * @param processSummary process summary logs.
+   * @param logger optionally provides a job logger
+   * in case one is not present.
+   */
+  async logProcessSummaryToJobLogger(
```

Comment:
I understand that this groups the logs and makes it easier to read but still 2 or more logs can be processed at almost the same time and make the logs intertwined, right? Wouldn't it be more useful to have a process id or something like that in a way the person who is analyzing can filter the logs and see only that is important for them. Also we don't have any date and time in the logs. I think that would be one of the most important things in the logs. What are your thoughts?

Reply:
This class is responsible for collecting the logs as far as they need to be grouped after some other class will write them. In the current PR our `LoggerService` is writing them to the console and the `job.logger` is writing them to the Bull dashboard.
Since it is just collecting them and not writing them there is no chance of an intertwined happening.
About the `date and time` it would be a good addition.
While writing to the `job.log` it can be added to the beginning of the line and while writing to the console we need to figure out where to add it to not conflict with the existing one.
Just keep in mind that this is mimicking what we have already in place and it does not have the date either. If there is nothing else major I will try to add the date right away.

Reply:
Ok. Thanks for the explanation!